benchexec/tools/esbmc.py

```diff
@@ -6,11 +6,11 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
+import re
 from benchexec.tools.sv_benchmarks_util import get_data_model_from_task, ILP32, LP64
 import benchexec.tools.template
 import benchexec.result as result
 
-
```

Comment:
Please keep this line (or format with `black`).

Reply:
Applied black